### ISSUE TITLE
Curses on windows

### ISSUE THREAD

**Issue by sznowak13** [sznowak13/super_snek#1]
Ok, so I tested the game on Windows and it totally blew colors away, printed some escape characters instead of actual colors. Curses was deigned to work with UNIX style terminals and Windows cmd is not one. The overall logic of the game is functionating but the colors are doomed.
I tested it with different color pairs - lower ones to be exact. First 16 color pairs are working fine, so I changed it in the config to match the first 16 and it works. 

TL;DR
I propose to change the mapping of the colors to not exceed 16:
COLORS = {
        'BLK': 0,
        'VLT': 14,
        'CYN': 12,
        'L_GRN': 11,
        'D_GRN': 3,
        'YLW': 15,
        'RED': 13,
        'TEAL': 10
    }
Should be now working on every Windows.
Thoughts? I'm a btit sleepy so the issue could be a mess gramatically.

**Comment by DavidGildour** [comment]
Did you try PowerShell? I would distinct color setup in main() so it determines whether its running on UNIX-based shell or windows.
BTW >> 'funcionating' << jebłem

**Comment by sznowak13** [comment]
'I'm a btit' rocks my world

**Comment by sznowak13** [comment]
Let's do the quick fix first so it is runnable from master, then let's make a branch out of that
